src/Discord.Net.Core/Entities/Guilds/IGuild.cs

```diff
@@ -117,5 +117,16 @@ public interface IGuild : IDeletable, ISnowflakeEntity
         Task DownloadUsersAsync();
         /// <summary> Removes all users from this guild if they have not logged on in a provided number of days or, if simulate is true, returns the number of users that would be removed. </summary>
         Task<int> PruneUsersAsync(int days = 30, bool simulate = false, RequestOptions options = null);
+
+        /// <summary> Gets a collection of all emotes in this guild. </summary>
+        Task<IReadOnlyCollection<GuildEmote>> ListEmotesAsync(RequestOptions options = null);
```

Comment:
Doesn't this already exist in the form of `IGuild.Emotes`?

Reply:
It also exists in the API. See: linked Discord API documentation.

Reply:
Is there an advantage to running `ListEmotesAsync` (which should probably be `GetEmotesAsync`) vs reading the cached collection?

Reply:
AFAIK, not really. But it exists, so I implemented it.

Reply:
A'ight, is commenting out acceptable? Will make it easier to re-add later.